contracts/core/Resolving/SportsAMMV2ResultManager.sol

```diff
@@ -94,26 +94,29 @@ contract SportsAMMV2ResultManager is Initializable, ProxyOwned, ProxyPausable, P
         int24 _line,
         ISportsAMMV2.CombinedPosition[] memory combinedPositions
     ) external view returns (bool isResolved) {
-        ResultType resultType = resultTypePerMarketType[_typeId];
-        if (resultType == ResultType.CombinedPositions) {
+        if (isGameCancelled[_gameId]) {
```

Comment:
The same check should add for `isMarketCancelled` method

Reply:
added